src/trapfunc.cpp

```diff
@@ -99,7 +99,7 @@ void trapfunc::cot( Creature *c, const tripoint & )
 void trapfunc::beartrap( Creature *c, const tripoint &p )
 {
     // tiny animals don't trigger bear traps
-    if( ( c != nullptr && c->get_size() == MS_TINY ) || c->is_hallucination() ) {
+    if( c != nullptr && c->get_size() == MS_TINY || c->is_hallucination() ) {
```

Comment:
I think that those parentheses were added for the same reason as for #30448.

Reply:
Ah yes sorry... you're totally right.